docs/operation.rst

```diff
@@ -4,3 +4,10 @@ Long-Running Operations
 .. automodule:: google.api_core.operation
   :members:
   :show-inheritance:
+
+Long-Running Operations in AsyncIO
+--------------
```

Comment:
(To fix the docs build)
```suggestion
-------------------------------------
```

Reply:
One of reasons why Markdown is more popular than RST...